source/github-events/on-fragment-load.ts

```diff
@@ -0,0 +1,10 @@
+import oneEvent from 'one-event';
+
+export default async function onFragmentLoad(fragment: HTMLElement | undefined, clickZone: HTMLElement): Promise<void> {
+	if (!fragment) {
+		return;
+	}
```

Comment:
`onFragmentLoad` makes it sound like it just waits for a fragment element to load, but actually it triggers its load through a behavior defined by the `details-menu` element.

In short, I'd like this method to be called `loadDetailsMenu` or something  like that and just receive the `details-menu` element. At that point, either the fragment and trigger the load, or just return